### ISSUE TITLE
Please add WebRTC support 

### ISSUE THREAD

**Issue by AceHack** [aspnet/SignalR#2387]
I would like to use SignalR as a mediator/rendezvous/fallback for WebRTC.  It would be great if I could facilitate direct peer to peer communications via signal r.

I realize this is a big enhancement.

Thanks for all your hard work.

**Comment by anurse** [comment]
We don't have any plans right now to enable peer-to-peer support using something like WebRTC. It's something we could keep in mind for the future, but we've got a pretty full queue of work already :). What's the specific scenario you think would work well here? Do you want clients to be able to call methods on other clients?

**Comment by AceHack** [comment]
Yes or in the least just have a sample that uses SignalR as a rendezvous server for WebRTC
